### DISCUSSION TITLE
Setup sponsorship / donations link

### DISCUSSION THREAD

**Discussion by funkypenguin** [rogerfar/rdt-client#484] [General]
Hey @rogerfar,

I'm sure many of us who enjoy RDTClient would like to contribute / donate, to support your efforts. 

I, for one, would like to do so, on behalf of [ElfHosted](https://elfhosted.com/open/sponsorship/), since RDTClient has become one of the most popular / critical apps for our Debrid-enjoying userbase.

Do you have a KoFi / GitHub Sponsors / something else setup that I we can contribute to, and point our users to?

Thanks!
D

**Comment by johnkennyuk** [comment]
Does make me wonder if donations / sponsors were accepted would that then entice more contributions by @rogerfar or other users via PR's, I only ask as im sure I read an issue reply from @rogerfar saying that this is a spare time project which I def understand but def could use some more Contributors with permissions to this repo to push the apps development further.  It does make me wonder why some high starred forkers of this repo haven't tried to PR there changes to the upstream??

_This post definitely isnt to criticize or put down as its still the best option for Servarr app integration with Debrid API's I just wanted to add to the thread with some thoughts as this app could be more than just best it could be great__

**Comment by rogerfar** [comment]
I appreciate the gesture, but at the moment it would not be fair to accept donations, I try to spend more time on it, but it's still a lower priority over everything else.

PR's I welcome for sure, but I also understand that the app is fairly complex and things like adding a new provider take significant amount of effort.

What I do require, if you want to add a new provider, I will need a test account with a multi month subscription so that I can actually develop it.
